test/Abp.ZeroCore.Tests/Zero/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -315,6 +369,128 @@ public void Should_Write_History_For_Audited_Entities_Update_Owned()
             _entityHistoryStore.Received().Save(Arg.Is<EntityChangeSet>(s => predicate(s)));
         }
 
+
+        [Fact]
+        public void Should_Write_History_For_Owned_Entities_Of_Audited_Entities_Create()
+        {
+            // Blog is the owner of BlogPromotion and has Audited attribute.
+            // Advertisement is not the owner of BlogPromotion.
+            // Therefore, BlogPromotion should follow Blog and have entity history.
+
+            int blog1Id;
+            using (var uow = Resolve<IUnitOfWorkManager>().Begin())
+            {
+                // Owned entities are not available via DbContext -> DbSet,
+                var blog1 = _blogRepository.Single(b => b.Name == "test-blog-1");
+                blog1Id = blog1.Id;
+                blog1.Promotions.Count.ShouldBe(0);
+
+                var advertisement1 = _advertisementRepository.Single(a => a.Banner == "test-advertisement-1");
+                blog1.Promotions.Add(new BlogPromotion { AdvertisementId = advertisement1.Id });
+                uow.Complete();
+            };
+
+            WithUnitOfWork(() =>
+            {
+                // Owned entities are not available via DbContext -> DbSet,
+                var blog1 = _blogRepository.Single(b => b.Name == "test-blog-1");
+                blog1.Promotions.Count.ShouldBe(1);
+            });
+
+            Predicate<EntityChangeSet> predicate = s =>
+            {
+                s.EntityChanges.Count.ShouldBe(1);
+
+                var entityChange1 = s.EntityChanges.Single(ec => ec.EntityTypeFullName == typeof(BlogPromotion).FullName);
+                entityChange1.ChangeType.ShouldBe(EntityChangeType.Created);
+                // The primary key of BlogPromotion is a shadow property,
+                // EF Core is keeping the values of PK of Blog and PK of BlogPromotion the same
```

Comment:
This is not true for collections of owned types.
You can verify that the next line of code fails if you add to `blog2.Promotions` instead.

From https://docs.microsoft.com/en-us/ef/core/modeling/owned-entities#collections-of-owned-types:

> when owned types are defined through a collection, it isn't enough to just create a shadow property to act as both the foreign key into the owner and the primary key of the owned instance, as we do for `OwnsOne`: there can be multiple owned type instances for each owner, and hence the key of the owner isn't enough to provide a unique identity for each owned instance.

Reply:
I see. Will take a look later.

Reply:
To put the scenario described in EF Core Owned Types documentation into our context.

`BlogPromotion` is configured with `Blog` as Owner with `BlogId` being the FK and with a PK named `Id` using the following
```csharp
b.WithOwner().HasForeignKey(bp => bp.BlogId);
b.Property<int>("Id");
b.HasKey("Id");
```
Therefore, when new `BlogPromotion` being added/created , the `Id` will be a running number and its FK `BlogId` will follow the PK of the owner (Blog).

Reply:
Yes.